subprojects/docs/src/docs/release/notes.md

```diff
@@ -54,6 +54,12 @@ In this release, plugin authors can use the `ObjectFactory.fileCollection()` met
 
 [The JaCoCo plugin](userguide/jacoco_plugin.html) has been upgraded to use [JaCoCo version 0.8.3](http://www.jacoco.org/jacoco/trunk/doc/changes.html) instead of 0.8.2 by default.
 
+## Better help message on delete operation failure
+
+The `:clean` task, all `Delete` tasks, and `project.delete {}` operations now provide with a better help message when failing to delete files. The most frequent and hard to troubleshoot causes for failing to delete files are other processes holding file descriptors open, and, concurrent writes.
+
+The help message now displays each failed path, handy to identify which process might be holding files, and also new paths found in a directory after failure, handy to spot concurrent writes.
```

Comment:
"...each failed path, which may helpful in identifying which process might be holding files open, and will also display any files that were created in the target directory after a delete failure, which may be helpful in identifying when a process is still writing to the directory."